ChangeLog.md

```diff
@@ -5,10 +5,10 @@ Note: This file only contains high level features or important fixes.
 ## 3.5
 
 ### 3.5.0 - Not yet released
-* 
+* Add ESTIMATOR_STATUS values to new estimatorStatus Vehicle FactGroup. These are now available to display in instrument panel.
```

Comment:
If all contributors would do this consistently then you inevitably have a lot of merge conflicts in this file, because everybody is changing stuff here.

Reply:
@LorenzMeier Is there some way to do it so that won't happen?